receiver/mongodbatlasreceiver/receiver.go

```diff
@@ -90,6 +90,7 @@ func (s *receiver) poll(ctx context.Context, time timeconstraints) (pdata.Metric
 		}
 		for _, project := range projects {
 			resourceAttributes.InsertString("mongodb.atlas.project", project.Name)
+			resourceAttributes.InsertString("mongodb.atlas.project.id", project.ID)
```

Comment:
This goes against the [naming guidelines](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-naming.md#attribute-naming)

> Names SHOULD NOT coincide with namespaces

Maybe `mongodb_atlas.project.name` and `mongodb_atlas.project.name`?

Reply:
Thanks @dmitryax, let me read through this again and update.